ex9/phonebook.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" href="style/style.css">
+  <script src="js/lib/jquery-3.1.1.js"></script>
+  <script src="js/lib/jquery.tmpl.js"></script>
+  <script data-main="js/main" src="js/lib/require.js"></script>
+</head>
+<body>
+  <input type="button" id="btnReloadList" value="Reload" />
+  <input type="button" id="btnASC" value="ASC" />
+  <input type="button" id="btnDESC" value="DESC" hidden />
+
+  <input type="button" id="btnSearchOn" value="Search" />
+  <input type="text" id="textSearch" value="" hidden />
```

Comment:
I would put all the search stuff into some container (`<div>`) so that I can hide it all in one action. It would also help to see the high-level features in your page.
You can then also give the container an ID/class (let's say "search") and rename the inner elements' ID, or might not need to give them IDs/classes at all as they are the only ones in this container (`#search input { ... }`).